src/components/SwapForm/Form.tsx

```
@@ -0,0 +1,307 @@
+            setOutputAmount('');
+            setOutputConvertedAmount('');
+            setInputAmount(amount);
+            return;
+        } else {
+            setInsufficientFunds(false);
+        }
+
+        setInputAmount(amount);
+
+        // If the input is empty or invalid, reset the converted and output amounts
+        if (isNaN(parseFloat(amount)) || amount === '') {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)